spec/controllers/v2/reports_controller_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe V2::ReportsController do
+  before do
+    stub_rbac_permissions(
+      Rbac::INVENTORY_HOSTS_READ,
+      Rbac::SYSTEM_READ,
+      Rbac::REPORT_READ
+    )
+  end
+
+  let(:attributes) do
+    {
+      title: :title,
+      os_major_version: :os_major_version,
+      ref_id: :ref_id,
+      description: :description,
+      profile_title: :profile_title,
+      business_objective: :business_objective,
+      system_count: -> { 0 }, # TODO
+      # TODO: compliant_system_count: -> { 0 },
+      # TODO: test_result_system_count: -> { 0 },
+      # TODO: unsupported_system_count: -> { 0 },
+      compliance_threshold: :compliance_threshold
+    }
+  end
+
+  let(:current_user) { FactoryBot.create(:v2_user) }
+  let(:rbac_allowed?) { true }
+
+  before do
+    request.headers['X-RH-IDENTITY'] = current_user.account.identity_header.raw
+    allow(StrongerParameters::InvalidValue).to receive(:new) { |value, _| value.to_sym }
+    allow(controller).to receive(:rbac_allowed?).and_return(rbac_allowed?)
+  end
+
+  context '/reports' do
+    describe 'GET index' do
+      let(:extra_params) { { account: current_user.account } }
+      let(:parents) { nil }
+
+      foreign = FactoryBot.create(:v2_account)
+      5.times do
+        FactoryBot.create(
+          :v2_report,
+          account: foreign
+        )
+      end
```

Comment:
You are creating DB entities outside tests, of course it is failing on duplication